samples/bluetooth/throughput/README.rst

```diff
@@ -87,6 +87,12 @@ You can use any two of the development kits listed above and mix different devel
        CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
        CONFIG_BT_RX_BUF_LEN=255
 
+  If you are using the Zephyr Bluetooth LE Controller, add the following option to the configuration of the application core sample:
+
+  .. code-block:: none
+
+       CONFIG_BT_DRIVER_QUIRK_NO_AUTO_DLE=y
+
```

Comment:
we will get rid of this dependency by using User Data Length update which is included here: https://github.com/nrfconnect/sdk-nrf/pull/3589
It would be easiest to just remove this now to avoid that PR having to remove this again.

Reply:
Removed the doc update commit